packages/Confirmation/stories/cypress.stories.js

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+
+import ConfirmationExample from "./examples/ConfirmationExample";
+
+storiesOf("confirmation / cypress", module).add("confirmation test", () => <ConfirmationExample />);
```

Comment:
I don't think you'll need a Cypress story for this component... I think `react-testing-library` is going to give you enough coverage... but maybe a Screener one would be good to have?